apps/frontpage/app/event-log/route.ts

```diff
@@ -131,6 +131,47 @@ export async function POST(request: NextRequest) {
     }
   }
 
+  const ip = headers.get('x-forwarded-for') ?? headers.get('x-real-ip');
+  const { userAgent, step } = received.payload ?? {};
+
+  if (userAgent) {
+    try {
+      let name = received.eventType;
+
+      // FIXME: want a more general way to handle this
+      if (step) {
+        name = `${name} - ${step}`;
+      }
+
+      const { builder, renderer, framework, storybookVersion } =
+        received.metadata ?? {};
+
+      const props = {
+        builder,
+        renderer,
+        framework: framework?.name,
+        storybookVersion,
+      };
+
+      await fetch('https://plausible.io/api/event', {
```

Comment:
Do we want to wait on this before sending the real event?

Maybe we should create an array of promises and then do `Promise.all()` at the end to do them in parallel?